src/users/users.service.ts

```diff
@@ -26,8 +33,30 @@ export class UsersService {
     return this.userRepository.save(user);
   }
 
+  async createTokenForUser(
+    userName: string,
+    identifier: string,
+  ): Promise<AuthToken> {
+    const user = await this.getUserByUsername(userName);
+    let accessToken = '';
+    for (let i = 0; i < 100; i++) {
```

Comment:
Do you want to hard-code these 100 tries to find a token?

Reply:
I thought what would be a good upper limit of tries. Maybe 10 would be better? I don't think this should configurable or anything.

Reply:
Well, we've got a chance of 1/2^512 collisions. I think we can safely remove this loop.